content/docs/glossary/cost-per-thousand.md

```diff
@@ -0,0 +1,15 @@
+---
+seo:
+  title: Cost per thousand (CPM)
+  description: CPM is used to measure how much 1000 advertisement impressions costs on one webpage
+  keywords: CPM, cost per thousand, web ads, advertisement, ads
+title: Cost per thousand
+weight: 0
+layout: page
+navigation:
+  show: false
+---
+
+Cost per thousand is a term used by marketers to indicate the price of 1,000 advertisement impressions on one webpage. If the owner of a website charges $3.00 CPM, an advertiser must pay $3.00 for every 1,000 impressions of their ad. The "M" in CPM stands for the word "mille," meaning "thousands" in Latin.
```

Comment:
"CPM, or Cost Per Thousand Impressions, is a term used to indicate the price of 1,000 impressions of an ad. For example, if the owner of a website charges a $3.00 CPM, an advertiser would pay $3.00 for every 1,000 impressions of their ad. The "M" in CPM stands for the word "mille," meaning "thousands" in Latin."

Just some small clarifications as it doesn't always have to happen on the same website.